client/src/components/J40Alert/tests/__snapshots__/j40Alert.test.tsx.snap

```diff
@@ -1,8 +1,20 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`rendering of the J40Alert tests the rendering of J40Alert 1`] = `
+exports[`rendering of the J40Alert tests the rendering of J40Alert with padding 1`] = `
 <DocumentFragment>
-  <div>
+  <div
+    class="undefined undefined"
```

Comment:
`"undefined undefined"` again.. same code generating?

Reply:
Yeah, this is actually the same PR as last time. Same issue has already been captured in a ticket